src/Forms/test/Xamarin.Forms.Xaml.UnitTests/Issues/Bz55343.xaml

```diff
@@ -5,7 +5,7 @@
 	<ContentPage.Resources>
 		<ResourceDictionary>
 			<OnPlatform x:Key="HeaderFont"
-                        x:TypeArguments="x:String">
+                        x:TypeArguments="NamedSize">
```

Comment:
This was originally a `Font` and the property type was also font. Now that the `Font` property is gone, I would expect to set the font size.